packages/template/webpack/tmpl/renderer.js

```diff
@@ -26,4 +26,6 @@
  * ```
  */
 
-console.log(`👋 This message is being logged by "renderer.js", included via webpack`);
+import url from './index.css';
```

Comment:
Unused import?

```suggestion
import './index.css';
```

Reply:
This is what I get for copy/pasting from the docs :laughing: